Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -208,8 +210,9 @@ function vtkRenderWindowInteractor(publicAPI, model) {
         model.container[invMethod]('mousemove', publicAPI.handleMouseMove);
       }
 
+      rootElm[method]('mouseenter', publicAPI.handleMouseEnter);
+      rootElm[method]('mouseleave', publicAPI.handleMouseLeave);
```

Comment:
They should not be added to the document.

Reply:
Removed them in `interactionRegistration`.